src/index.html

```diff
@@ -26,10 +26,14 @@
     <!-- ! DO NOT REMOVE THIS COMMENT, WE NEED ITS CLOSING MARKER -->
   </head>
 
-  <body>
+  <body style="background: url('./src/assets/img/buddhanexus.jpg') cover;">
+    <div class="head">
```

Comment:
naming, nitpicking: `head` as a class name is a bit confusing, since there exists a `head` element in HTML already. I would pick a different name - maybe `logo-container`?